src/liquid/components/ld-header/readme.md

```diff
@@ -0,0 +1,306 @@
+---
+eleventyNavigation:
+  key: Header
+  parent: Components
+layout: layout.njk
+title: Header
+permalink: components/ld-header/
+---
+
+<link rel="stylesheet" href="css_components/ld-header.css">
+<link rel="stylesheet" href="css_components/ld-button.css">
+<link rel="stylesheet" href="css_components/ld-icon.css">
+<link rel="stylesheet" href="css_components/ld-typo.css">
+
+# ld-header
+
+The `ld-header` component serves as a wrapper for creating a fully customizable page header. You can add whatever content you want. The component only takes care of the following things regarding its content:
+
+* It adds a space between all its direct children.
+* It takes care of a proper styling of contained `ld-button` elements.
+
+Also, you can easily make the header sticky and make it hide when the user is scrolling down by using the `sticky` and `hide-on-scroll` props.
+
+---
+
+## Examples
+
+### Default
+
+{% example %}
+<ld-header>
+  <ld-typo class="logo" tag="div" variant="b6" title="Logo">M</ld-typo>
+  <ld-typo tag="div" variant="h5">Liquid Oxygen</ld-typo>
+</ld-header>
+
+<style>
+  .logo {
+    color: var(--ld-thm-warning);
+    margin-top: calc(var(--ld-sp-8) * -1);
```

Comment:
The `<title>` in the SVG will behave like the `title`-attribute and show up like a little tooltip. Not sure if we want that here?

Instead of adding an SVG, I'd prefer using a component here, as the examples are already bloated enough. Will use the `ld-icon` component here with additional styling for the size.

Reply:
Yes, let's use the `ld-icon` component with the `aria-label` attribute.

Reply:
Added this to the anchor / surrounding div-tag so that the prop also applies to custom logos. Is that okay?

Reply:
Also, added "Merck KGaA, Darmstadt, Germany" as default label for non-custom-logos.